### PR TITLE
local command scaffolding and config

### PR DESCRIPTION
**Pull request by scott-cotton**
stacked on #41 

To be updated after discussion from todays meeting with new names for things in CLI scaffolding

### CODE REVIEW

File: internal/command/localdaemon/locald.go
Comment (daniel-de-vera) [inline internal/command/localdaemon/locald.go:1]: I would rename `localdaemon` to `locald` (that way it matches with the subcommand and also the package under internal).

File: internal/command/command.go [hunk 1]
```diff
@@ -34,6 +35,7 @@ func New() *cobra.Command {
 		sandbox.New(cfg),
 		routegroup.New(cfg),
 		resourceplugin.New(cfg),
+		local.New(cfg),
```
Comment (daniel-de-vera) [inline internal/command/command.go:38]: Why not making `localdaemon.newLocalDaemon()` public and add it in here?

**Review by daniel-de-vera — APPROVED** [review]
LGTM

**Comment by scott-cotton** [comment]
note: will make the change to use the flag --sandbox-manager in another PR

**Pull request merged**